### ISSUE TITLE
Feature request:  separate incoming/outgoing whitelist

### ISSUE THREAD

**Issue by akhepcat** [pavel-odintsov/fastnetmon#444]
add an optional "ignore_(incoming|outgoing)" flag to each CIDR block in the /etc/networks_whitelist

this would allow high-speed one-way clients to still have protection in the opposite direction.

**Comment by pavel-odintsov** [comment]
Hello!

If you are interested in netflow or sflow you could wrote own hook script https://github.com/pavel-odintsov/fastnetmon/blob/master/src/netflow_hooks.lua or https://github.com/pavel-odintsov/fastnetmon/blob/master/src/sflow_hooks.lua and filter out some packets according to source or destination IP or direction.

**Comment by cuddylier** [comment]
I would also be interested in this as I have some servers on a ddos protected network but sometimes an attack can leak for a few seconds but I would want to avoid them being nullrouted as usually the large network port copes fine so would only want outbound to be filtered to stop any outbound abuse.

**Comment by pavel-odintsov** [comment]
Do you need it for port mirror?

**Comment by cuddylier** [comment]
I would just be using sflow so whatever that is, I don't think port mirroring?

**Comment by pavel-odintsov** [comment]
Yes. So you could use mentioned earlier approach and parse data with LUA :)

**Comment by akhepcat** [comment]
Never used LUA, actually. 

is there an example available in the repo?

**Comment by pavel-odintsov** [comment]
I have shared two links above :)

**Comment by akhepcat** [comment]
Ah, those are examples, and not the hook interface.   Sorry, naming got me.

**Comment by pavel-odintsov** [comment]
Yes it's example script for lua ;)
